internal/kptfile/pkgfile_test.go

```diff
@@ -240,6 +308,50 @@ openAPI:
                 setter:
                     name: tag
                     value: 1.8.0
+`,
+		},
+		{
+			name: "diverged to and from",
```

Comment:
If they don't already exist, add test cases for the following:

- updated changed from original, to contains the value in original
- updated changed from original, to contains a different value from original
- update changed from original, to does not contain the value
- updated deleted from original, to contains the value in original
- updated deleted from original, to contains a different value from original

Reply:
Done.